test/server/cards/06-WoE/TyaArhiEsquire.spec.js

```diff
@@ -0,0 +1,48 @@
+describe('TyaArhiEsquire', function () {
+    describe("TyaArhiEsquire's ability", function () {
+        beforeEach(function () {
+            this.setupTest({
+                player1: {
+                    house: 'ekwidon',
+                    token: 'grunt',
+                    inPlay: ['antiquities-dealer', 'grunt:mass-buyout', 'tya-arhi-esquire'],
```

Comment:
I think using `mass-buyout` here and at line 9 can be confusing since you are using mass buyout for board wipe.

Reply:
Fixed, thanks!